tests/integration/test_api/test_config/test_limit_eps/data/test_cases/limit_eps_config.yaml

```diff
@@ -0,0 +1,161 @@
+- name: UPLOAD_ONE_BLOCK_CONFIG_NOT_ALLOWED
+  description: Upload one block configuration
+  configuration_parameters:
+    ALLOWED: no
+  metadata:
+    expected_code: 200
+    expected_error_code: 1
+    expected_max_value: 100
+    expected_timeframe_value: 10
+    request_body: >
+      <ossec_config>
+        <remote>
+          <connection>secure</connection>
+          <port>1514</port>
+        </remote>
+        <global>
+          <limits>
+            <eps>
+              <maximum>500</maximum>
+              <timeframe>50</timeframe>
+            </eps>
+          </limits>
+        </global>
+      </ossec_config>
+
+- name: UPLOAD_ONE_BLOCK_CONFIG_ALLOWED
+  description: Upload ones block configuration
```

Comment:
```suggestion
  description: Upload one block configuration
```

Reply:
Changed in commit 55c3deb54b1205546430149e7a80b348833da665.